File: testdata/tasks/envido/graders/grader.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int envido(int numero1, string &palo1, int numero2, string &palo2, int numero3, string &palo3);

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    int numero1;
    cin >> numero1;
    string palo1;
    cin >> palo1;
    int numero2;
    cin >> numero2;
    string palo2;
    cin >> palo2;
    int numero3;
    cin >> numero3;
    string palo3;
    cin >> palo3;
    int returnedValue;
    returnedValue = envido(numero1, palo1, numero2, palo2, numero3, palo3);
    cout << returnedValue << "\n";
    return 0;
}

File: testdata/tasks/envido/graders/grader.java
import java.io.*;
import java.lang.*;
import java.util.*;

public class grader {
    private static String nextLine = "";
    private static int nextIndex = 0;
    private static BufferedReader reader;
    
    private static String readToken() throws IOException {
        while (true) {
            while (nextIndex < nextLine.length() && nextLine.charAt(nextIndex) == ' ') nextIndex++;
            if (nextIndex == nextLine.length()) {
                nextLine = reader.readLine();
                nextIndex = 0;
            } else {
                break;
            }
        }
        int baseIndex = nextIndex++;
        while (nextIndex < nextLine.length() && nextLine.charAt(nextIndex) != ' ') nextIndex++;
        return nextLine.substring(baseIndex, nextIndex);
    }
    
    public static void main(String [] args) throws IOException {
        reader = new BufferedReader(new InputStreamReader(System.in));
        try (PrintWriter writer = new PrintWriter(new BufferedWriter(new OutputStreamWriter(System.out)))) {
            int numero1;
            numero1 = Integer.parseInt(readToken());
            String palo1;
            palo1 = (readToken());
            int numero2;
            numero2 = Integer.parseInt(readToken());
            String palo2;
            palo2 = (readToken());
            int numero3;
            numero3 = Integer.parseInt(readToken());
            String palo3;
            palo3 = (readToken());
            int returnedValue;
            returnedValue = envido.envido(numero1, palo1, numero2, palo2, numero3, palo3);
            writer.println(returnedValue);
        }
    }
}
